web/src/components/Header/Header.scss

```diff
@@ -45,13 +53,22 @@
         .logo {
           padding-right: 0.75rem;
           padding-left: 0.75rem;
-          background-color: rgba(255, 255, 255, 0.9);
           border-radius: 0.625rem;
           display: flex;
           align-items: center;
           margin-right: 5px;
           height: 3rem;
 
+          &.light {
+            // !DARK: not sure if this is the preferred color
```

Comment:
!DARK ?

Reply:
its just the pattern ive chosen to leave comments related to this branch. the fact that the comment says `DARK` inside of the .light selector means nothing